src/arm/arm/arm.py

```diff
@@ -0,0 +1,85 @@
+import sys
+sys.path.append("/home/trickfire/urc-2023/src")
+import rclpy
+from rclpy.node import Node
+from rclpy.executors import ExternalShutdownException
+from enum import IntEnum
+from custom_interfaces.srv import ArmMode
+from std_msgs.msg import Int32
+from .individual_control_vel import IndividualControlVel
+from interface.robot_interface import RobotInterface
+
+
+
+from utility.color_text import ColorCodes
+
+
+class ArmModeEnum(IntEnum):
+    disabled = 0
+    individual_motor_control_vel = 1
+    individual_motor_control_pos = 2
+    inverse_kinematics = 3
+    
+
+class Arm(Node):
+    def __init__(self):
+        super().__init__("arm_node")
+        self.get_logger().info(ColorCodes.BLUE_OK + "Launching arm_node" + ColorCodes.ENDC)
+        
+        self.changeArmModeSub = self.create_subscription(Int32, "update_arm_mode", self.updateArmMode, 10)
+        
+        self.current_mode = ArmModeEnum.disabled
+        
+        self.mode_service = self.create_service(ArmMode, "get_arm_mode", self.mode_serviceHandler)
+        
+        self.botInterface = RobotInterface(self)
+        
+        self.individualControlVel = IndividualControlVel(self, self.botInterface)
+        
+    def mode_serviceHandler(self, request, response):
+        response.current_mode = int(self.current_mode)        
+        return response
+    
+    def updateArmMode(self, msg):
+        self.current_mode = msg.data
+        
+        if self.current_mode == 0:
+            self.botInterface.disable_arm_turntable_motor()
+            self.botInterface.disable_arm_shoulder_motor()
+            self.botInterface.disable_arm_elbow_motor()
+            self.botInterface.disable_arm_left_wrist_motor()
+            self.botInterface.disable_arm_right_wrist_motor()
+            
+            self.individualControlVel.canSend = False
+        elif (self.current_mode == 1):
+            self.individualControlVel.canSend = True
+        elif (self.current_mode == 2):
+            self.individualControlVel.canSend = False
+        elif (self.current_mode == 3):
+            self.individualControlVel.canSend = False
```

Comment:
im not a huge fan of the way mode switching is done, but im gonna be real i cant think of a better way right now, so we can just put a pin in that.